lib/caps/TGL/iHD

```diff
@@ -0,0 +1,81 @@
+###
+### Copyright (C) 2019 Intel Corporation
```

Comment:
Will we consider roll 2019 to 2020 ?

Reply:
Zachary, I am fine on this. 
@uartie Artie, how do you think?